products/wordpress.md

```diff
@@ -214,8 +221,13 @@ is not guaranteed. [Version 3.7 to 4.0 won't get security updates after Dec 1, 2
 
 ## [PHP Support](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/)
 
+:warning: Note that PHP 8 compatibility is 
+[considered as beta](https://make.wordpress.org/core/2020/11/23/wordpress-and-php-8-0/),
+and thoroughly testing a site is recommended before upgrading it to PHP 8.
+
 | WordPress Version  | Supported PHP Versions |
 ---------------------|------------------------|
+| 6.2                | 5.6 - 8.2              |
```

Comment:
Used `table.html` (for the first time 🎉!)

Reply:
Great ! Will update the `collapsedCycles` expression to exclude 3.6 as it does not declare a `supportedPHPVersions`.

Note that it could be done in the `table.html` include but I found it more practical to do it explicitly : that makes things easier to spot missing declarations.